### PR TITLE
docs(tutorial): easy deploy tutorials

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+content:
+  h1: Deploying WordPress in a Kubernetes cluster with the Application Library
+  paragraph: Learn how to deploy WordPress on Kubernetes using the Easy Deploy feature in a few clicks
+tags: kubernetes kubernetes-kapsule wordpress container-registry easy-deploy deploy containerized-application docker
+dates:
+  validation: 2022-04-07
+  posted: 2022-04-07
+---
+
+WordPress is a well-known CMS (content management system) allowing you to create evolving websites.
```
Comment (nedaehm) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:18]:
```suggestion
WordPress is a well-known CMS (content management system) that allows you to create evolving websites.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+This tutorial teaches you how to install WordPress on a Scaleway [Kubernetes Kapsule](https://www.scaleway.com/en/docs/compute/kubernetes/quickstart/) or Kosmos cluster by using the [Application Library in the Easy-Deploy feature](https://www.scaleway.com/en/docs/compute/kubernetes/how-to/enable-easy-deploy/). 
+You can achieve most of this directly from within the Scaleway Console. The steps of this tutorial are shown in the side menu on the right.
```
Comment (nedaehm) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:21]:
```suggestion
You can achieve most of this directly from the Scaleway Console. The steps of this tutorial are shown in the side menu on the right.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+categories:
+  - compute
+  - kubernetes
+  - database
+meta:
+  title: Deploying WordPress in a Kubernetes cluster with the Application Library
+  description: Learn how to deploy WordPress on Kubernetes using the Easy Deploy feature in a few clicks
+content:
+  h1: Deploying WordPress in a Kubernetes cluster with the Application Library
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:10]:
```suggestion
  h1: Deploying WordPress on a Kubernetes cluster with the Application Library
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
+---
+categories:
+  - compute
+  - kubernetes
+  - database
+meta:
+  title: Deploying WordPress in a Kubernetes cluster with the Application Library
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:7]:
```suggestion
  title: Deploying WordPress on a Kubernetes cluster with the Application Library
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 2 above)
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:21]:
```suggestion
You can perform most of these steps directly from within the Scaleway Console. 
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  - You have a [Kubernetes cluster](https://www.scaleway.com/en/docs/compute/kubernetes/how-to/create-cluster/) at Scaleway (Kapsule or Kosmos)
+  - You have [kubectl](https://www.scaleway.com/en/docs/compute/kubernetes/how-to/connect-cluster-kubectl) installed on your local computer
+
+</Message>
+
+## Creating a Kubernetes Kapsule Cluster and deploying WordPress
+
+1. Go to the [Kubernetes dashboard](https://console.scaleway.com/kapsule/clusters) in the Scaleway console.
+2. Select the cluster you want to deploy WordPress to.
+3. Select the Easy Deploy tab.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:35]:
```suggestion
3. Select the **Easy Deploy** tab.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+    <Lightbox src="scaleway-kubernetes-easy-deploy.webp" alt="" />
+
+    The Application Library gives us the possibility to deploy a WordPress CMS application, which we can find by using the search bar on our Scaleway Console.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:39]:
```suggestion
    The Application Library allows you to deploy a WordPress CMS application. You can use the search bar to locate the application. 
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 1 above)
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:18]:
```suggestion
WordPress is a well-known CMS (**C**ontent **M**anagement **S**ystem) allowing you to create evolving websites.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+6. Search for the application you want to deploy using the search bar.
+
+    <Lightbox scr="scaleway-kubernetes-wordpress-app.webp" alt="" />
+
+    By selecting it, the default configuration for the selected application will be set automatically in a [YAML](https://yaml.org) format.
+    You can edit the configuration to fit to your application deployment within the YAML editor displayed.
+
+    <Lightbox src="scaleway-kubernetes-yaml-editor.webp" alt="" />
+
+7. Enter a name for your application as well as a Kubernetes namespace to deploy it to.
```
Comment (nedaehm) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:55]:
```suggestion
7. Enter a name for your application and the Kubernetes namespace you want to deploy it to.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+1. Go to the [Kubernetes dashboard](https://console.scaleway.com/kapsule/clusters) in the Scaleway console.
+2. Select the cluster you want to deploy WordPress to.
+3. Select the Easy Deploy tab.
+
+    <Lightbox src="scaleway-kubernetes-easy-deploy.webp" alt="" />
+
+    The Application Library gives us the possibility to deploy a WordPress CMS application, which we can find by using the search bar on our Scaleway Console.
+
+4. Click **Deploy an application**.
+5. Select **Application Library** as application type.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:42]:
```suggestion
5. Select **Application Library** as the application type.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 6 above)
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:39]:
```suggestion
    The Application Library gives you the possibility to deploy a WordPress CMS application, which you can find by using the search bar on the Scaleway console.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+    By selecting it, the default configuration for the selected application will be set automatically in a [YAML](https://yaml.org) format.
+    You can edit the configuration to fit to your application deployment within the YAML editor displayed.
+
+    <Lightbox src="scaleway-kubernetes-yaml-editor.webp" alt="" />
+
+7. Enter a name for your application as well as a Kubernetes namespace to deploy it to.
+
+    <Message type="important">
+
+    If no namespace is chosen, your application will be deployed in your cluster’s `default` namespace. Deploying in the `default` namespace is not a good practice and it is highly recommended to split your deployments, services, and application in separate namespace, thus allowing a more secure configuration of your cluster.
```
Comment (nedaehm) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:59]:
```suggestion
    If you do not choose a namespace, your application will be deployed in your cluster’s `default` namespace. Deploying in the `default` namespace is not a good practice and it is highly recommended to split your deployments, services, and application in separate namespace, thus allowing a more secure configuration of your cluster.
```

Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:59]:
```suggestion
    If no namespace is chosen, your application will be deployed in your cluster’s `default` namespace. Deploying in the `default` namespace is not a good practice and it is highly recommended to split your deployments, services, and application in separate namespaces, thus allowing a more secure configuration of your cluster.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+4. Click **Deploy an application**.
+5. Select **Application Library** as application type.
+
+    <Lightbox src="scaleway-kubernetes-deploy-app.webp" alt="" />
+
+6. Search for the application you want to deploy using the search bar.
+
+    <Lightbox scr="scaleway-kubernetes-wordpress-app.webp" alt="" />
+
+    By selecting it, the default configuration for the selected application will be set automatically in a [YAML](https://yaml.org) format.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:50]:
```suggestion
    The default configuration for the application is set automatically in a [YAML](https://yaml.org) format when you select it.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+    You are then redirected to the listing page of our deployed applications.
+
+    <Lightbox src="scaleway-kubernetes-deployed-apps-list.webp" alt="" />
+
+9. Check that the deployment is successful by running the command `kubectl get all -n cms-ns` in your terminal.
+
+    <Lightbox src="scaleway-kubernetes-kubectl-all.webp" alt="" />
+
+    You can see that your WordPress application is running, and even has created a mariadb database within your cluster to store our data on Scaleway Block Storage.
```
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:75]:
```suggestion
    You can see that your WordPress application is running, and has even created a mariadb database within your cluster to store our data on Scaleway Block Storage.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+    You also have access to the Load Balancer that was created to access our WordPress using the external IP displayed.
```
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:76]:
```suggestion
    You also have access to the Load Balancer that was created to access your WordPress using the external IP displayed.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 11 above)
Comment (nedaehm) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:75]:
```suggestion
    You can see that your WordPress application is running, and has even created a mariadb database within your cluster to store our data on Scaleway Block Storage.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+10. Access your WordPress application by using this external IP in our web browser. The WordPress website displays in your browser.
```
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:78]:
```suggestion
10. Access your WordPress application by using this external IP in your web browser. The WordPress website displays in your browser.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+
+    <Lightbox src="scaleway-kubernetes-wordpress.webp" alt="" />
+
+    <Message type="note">
+
+    * When wanting to update the configuration of our application, you may run into some errors returned by the API. Read these errors carefully, as they contain the instructions you need to solve them.<br />
```
Comment (RoRoJ) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:84]:
```suggestion
    * When wanting to update the configuration of your application, you may run into some errors returned by the API. Read these errors carefully, as they contain the instructions you need to solve them.<br />
```

Comment (RoRoJ) [reply tutorials/deploy-wordpress-from-application-library/index.mdx:84]: Is the <br />  meant to be there?

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 11 above)
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:75]:
```suggestion
    You can see that your WordPress application is running, and that it has created a Mariadb database within your cluster to store our data on Scaleway Block Storage.
```
add a link to block storage?

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 12 above)
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:76]: this sentence is a bit confusing

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 13 above)
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:78]:
```suggestion
10. Access your WordPress application by entering the external IP provided in a web browser. The WordPress website displays in your browser.
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx (hunk 14 above)
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:84]:
```suggestion
    * When updating the configuration of your application, you may run into some errors returned by the API. Read these errors carefully, as they contain the instructions you need to solve them.<br />
```

File: tutorials/deploy-wordpress-from-application-library/index.mdx [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+    * When wanting to update your WordPress application configuration, you run into an error asking you to use the correct password to have the authorization to update the application.<br /><br />
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-wordpress-from-application-library/index.mdx:85]:
```suggestion
    * When you update your WordPress application setup, you will run into an error that asks you to use the correct password to update the application.<br /><br /> 
```

**Review by RoRoJ — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**